app/databrowser/src/main/java/org/phoebus/archive/reader/appliance/ApplianceValueIterator.java

```diff
@@ -313,4 +313,61 @@ protected static AlarmSeverity getSeverity(int severity) {
             return AlarmSeverity.UNDEFINED;
         }
     }
+
```

Comment:
The alarm status text to index mappings are defined in EPICS base, the C code running on the IOC.
For Java, the closest we have is in the Channel Access client library https://github.com/epics-base/jca/blob/master/src/core/gov/aps/jca/dbr/Status.java.

I would call  `gov.aps.jca.dbr. Status. forValue(int status)` instead of basically duplicating that in here.